debian/zookeeper/Dockerfile.rpm

```diff
@@ -29,7 +29,11 @@ ENV COMPONENT=zookeeper
 
 RUN echo "===> installing ${COMPONENT}..." \
     && yum -q -y update \
-    && yum install -y confluent-kafka-${SCALA_VERSION}-${KAFKA_VERSION} \
+    && if [ "x${ALLOW_UNSIGNED}" = "xtrue" ]; then \
+        yum install -y --nogpgcheck confluent-kafka-${SCALA_VERSION}-${KAFKA_VERSION}${CONFLUENT_PLATFORM_LABEL}; \
```

Comment:
Seems like we could avoid most of these additions/variants if we just setup the yum repo config with the flag set correctly. The setup of the file is a bit messy currently, but if it's going to get refactored anyway, might be better to have that localized to one place.